test/unit/views/DisassociateUcrViewSpec.scala

```diff
@@ -96,6 +96,12 @@ class DisassociateUcrViewSpec extends ViewSpec with Injector {
       }
     }
 
+    "form contains 'MUCR' with value" in {
+      val mucrView = createView(DisassociateUcr.form.fill(DisassociateUcr(Mucr, ducr = None, mucr = Some("1234"))))
+      mucrView.getElementsByAttributeValue("for", "mucr").first() must containMessage("mucrOptions.mucr.label")
+      mucrView.getElementsByAttributeValue("for", "ducr").first() must containMessage("mucrOptions.ducr.label")
+    }
+
```

Comment:
Should there be an update to the following tests as well? ("form contains 'DUCR' with value")

Reply:
Done.